packages/workorder-angular/lib/services/api-service.js

```diff
@@ -0,0 +1,67 @@
+var CONSTANTS = require('../constants');
+
+function WorkorderApiService() {
+};
+
+WorkorderApiService.prototype.readWorkorder = function readWorkorder(workorderId) {
+  return;
+};
+
+WorkorderApiService.prototype.createWorkorder = function createWorkorder(workorderToCreate) {
+  return;
+};
+
+WorkorderApiService.prototype.begin = function begin(workorder) {
+  return;
+};
+
+WorkorderApiService.prototype.updateWorkorder = function updateWorkorder(workorderToUpdate) {
+  return;
+};
+
+
+WorkorderApiService.prototype.removeWorkorder = function removeWorkorder(workorderToRemove) {
+  return;
+};
+
+WorkorderApiService.prototype.listWorkflows = function listWorkflows() {
+  return;
+};
+
+WorkorderApiService.prototype.readWorkflow = function readWorkflow(workflowId) {
+  return;
+};
+
+
+WorkorderApiService.prototype.listResults = function listResults() {
+  return;
+};
+
+
+WorkorderApiService.prototype.readUser = function readUser(userId) {
+  return;
+};
+
+WorkorderApiService.prototype.listUsers = function listUsers() {
+  return;
+};
```

Comment:
Good question: this will be replaced with the code we will wrote for WFM and security.
I can mock that but I do not see much sense for now to do it considering that we should be ready to put something here really soon.